auraed/src/cells/cell_service/cell_service.rs

```diff
@@ -237,6 +241,29 @@ impl CellService {
         executables.broadcast_stop().await;
         Ok(())
     }
+
+    #[tracing::instrument(skip(self))]
+    async fn list(&self) -> Result<CellServiceListResponse> {
+        let cells = self.cells.lock().await;
+
+        
+        Ok(CellServiceListResponse { 
+            cells: cells.entries()
```

Comment:
This block could benefit from being formatted with rustfmt. If you are using VSCode you should be able to set it to run on save (https://stackoverflow.com/questions/67859926/how-to-run-cargo-fmt-on-save-in-vscode)